### PR TITLE
WRO-2823: Update deprecate message in `ui/GridListImageItem` to version 5.0.0

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated deprecate message in `ui/GridListImageItem` to version 5.0.0.

### Resolution
- Deprecate `ui/GridListImageItem` to be removed in 5.0.0.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-2823

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,4 +27,4 @@
     "eslint-config-enact-proxy": "^1.0.4",
     "lerna": "^4.0.0"
   }
-}
```
Comment (daniel-stoian-lgp) [inline package.json:30]: please revert this

File: packages/ui/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Deprecated
+
+- `ui/GridListImageItem` to be removed in 5.0.0
```
Comment (daniel-stoian-lgp) [inline packages/ui/CHANGELOG.md:9]:
```suggestion
- `ui/GridListImageItem` to be removed in 5.0.0, use `ui/ImageItem` instead
```

**Review by vladut-clapou-lgp — APPROVED** [review]

File: packages/ui/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 ### Deprecated
 
+- `ui/GridListImageItem` to be removed in 5.0.0, use `ui/ImageItem` instead
```
Comment (juwonjeong) [inline packages/ui/CHANGELOG.md:9]:
```suggestion
- `ui/GridListImageItem`, to be removed in 5.0.0. Use `ui/ImageItem` instead.
```

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**